src/main/java/ChargeRange/ChargeRangeApplication.java

```diff
@@ -0,0 +1,54 @@
+package ChargeRange;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ChargeRangeApplication {
+
+    private static boolean isAdjacentNumbersInSameRange(int reading1, int reading2) {
```

Comment:
is third condition possible due to sorting of values?

Reply:
In current implementation the third condition is not possible, but in future if there is some additional function/sort order change happens, we can reuse this function to check if values are adjacent. So I thought of adding it. Is it not required?

Reply:
It's a good thinking to think about future, and you are thinking in that direction 👍
We should think from design angle if this can be extended and make provision to extend easily and not make  code changes already ☺️
If you plan to send 1,2 or 3,4 now , it is sorted in ascending order. In case in future if you need to check sequence for both ascending and descending, then this is not the only place for change, isn't it... You should also think about all probable code changes. 
In such a scenario then 2 methods i.e. is ascending() and isDescending() might be more readable.
Choice is yours....
If you choose to keep the code then you need to add a test case and if you can't write a test case now, then it is unreachable code 😊

Reply:
Thank you for the feedback 👍 . I have removed the unreachable condition.